src/plone/restapi/services/navigation/get.py

```diff
@@ -29,7 +30,10 @@ def __init__(self, context, request):
 
     def __call__(self, expand=False):
         if self.request.form.get("expand.navigation.depth", False):
-            self.depth = int(self.request.form["expand.navigation.depth"])
+            try:
+                self.depth = int(self.request.form["expand.navigation.depth"])
+            except ValueError as e:
```

Comment:
```suggestion
            except (ValueError, TypeError) as e:
```

(Unlikely, but could be TypeError if the querystring had something like `expand.navigation.depth:list=asdf`)